### PR TITLE
prepares treemap for release

### PR DESCRIPTION
**Pull request by smurphy8**
budget_screen

fixed links to zoomable

visualizations

dropped 'under-construction' tag

dropped more oakland references

picture for timeline thumb

link to the wiki

### CODE REVIEW

**Comment by smurphy8** [comment]
Tried to do the squashing ahead of time this time!

**Comment by smurphy8** [comment]
@mascott @mkchandler @cahiller  If this move looks good, it would be nice if it could be merged.
I would like to use this move to factor out some of the functions being used to parse and filter the data into another js file so I can depend on them in my radar branch.

**Comment by mkchandler** [comment]
@smurphy8 I will try to take a look at this today or tomorrow.

**Comment by smurphy8** [comment]
:+1:  Thanks!

File: sandbox/zoomable.html [hunk 1]
```diff
@@ -74,7 +74,7 @@
 
 
 window.addEventListener('message', function(e) {
-    var opts = e.data.opts,
+    fvar opts = e.data.opts,
```
Comment (mkchandler) [inline sandbox/zoomable.html]: @smurphy8 was this supposed to be changed to `fvar`?

Comment (smurphy8) [inline sandbox/zoomable.html]: fixed, I think this came back when I squashed everything.  Sorry about that.

**Comment by mkchandler** [comment]
@smurphy8 Looks good :+1: I'm going to merge.

**Pull request merged**